### PR TITLE
Change timeout / wait settings on HF Helper

### PR DESCRIPTION
**Pull request by dkolas**
- Set default timeout to 30 sec instead of 10
- Allow passing timeout per usage
- Don't tell HF to wait on the model per call

### CODE REVIEW

**Review by EniasCailliau — APPROVED** [review]
Thanks for putting in the effort to get huggingface to work smoothly. Made a small suggestion regarding HF's estimated model load time.

File: src/steamship/utils/huggingface_helper.py [hunk 1]
```diff
@@ -52,9 +52,13 @@ async def _model_call(session, text: str, api_url, headers, additional_params: d
...
 async def _model_calls(
-    texts: List[str], api_url: str, headers, additional_params: dict = None
+    texts: List[str],
+    api_url: str,
+    headers,
+    timeout_seconds: int,
+    additional_params: dict = None,
 ) -> List[list]:
-    async with aiohttp.ClientSession(timeout=ClientTimeout(total=10)) as session:
+    async with aiohttp.ClientSession(timeout=ClientTimeout(total=timeout_seconds)) as session:
```
[the remark this answers is not in the document]
Comment (eob) [reply src/steamship/utils/huggingface_helper.py:61]: That's pretty nice of them to return that -- we should remember that for when we're in a similar spot.

**Review by eob — APPROVED** [review]
LGTM; Agree that Enias' idea might be worth trying

File: src/steamship/utils/huggingface_helper.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dkolas) [reply src/steamship/utils/huggingface_helper.py:61]: It does estimate the time to load the model, but in my experience thusfar, it has no relation to how long it actually takes to load the model.  That's why the _model_call method just waits 1 sec and retries.  

It also returns the same message when it simply doesn't have current capacity, so the message cannot be trusted, sadly.

The timeout here is for the execution of a specific request.

**Pull request merged**